### ISSUE TITLE
Doesn't work in Lucee 5.x

### ISSUE THREAD

**Issue by redtopia** [bennadel/JSONWebTokens.cfc#7]
I didn't spend a lot of time on this, but I got it working by scoping global variables in the CFCs.

Also it might be worth noting that real-world examples would likely require quotes around the payload properties.

Hit me up if you're interested in knowing what I did. Otherwise, if this project isn't really needed anymore and a different library would be better, adding that to the docs would be great.

**Comment by redtopia** [comment]
I did find this one, which works in Lucee out of the box. https://github.com/coldfumonkeh/cf_jwt

**Comment by 4strid** [comment]
Yeah adding explicit scope to the global variables fixed it for me.
